orte/orted/orted_main.c

```diff
@@ -300,6 +301,14 @@ int orte_daemon(int argc, char *argv[])
     opal_unsetenv(OPAL_MCA_PREFIX"ess", &orte_launch_environ);
     opal_unsetenv(OPAL_MCA_PREFIX"pmix", &orte_launch_environ);
 
+    /* We need to parse the MCA parameters to read the debug MCA
+     * parameters used by the plm to relay the debug options.
```

Comment:
This call is already done in orte_init, so this doesn't actually do anything as the function protects itself against registering the params twice

Reply:
But orte_init is called further down in the file on line 365 after we do the check on line 357 if we should detach from the controlling terminal. So we need to process the MCA parameters before that point so we can read the MCA parameter set my the PLM when relaying the `--debug-daemons` CLI option.

Reply:
Okay - doesn't hurt anything.